path_config.go

```diff
@@ -255,10 +262,13 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 	if config.JWKSURL != "" {
 		methodCount++
 	}
+	if len(config.JWKSPairs) > 0 {
```

Comment:
I left the `jwks_url` and `jwks_ca_pem` config in place to maintain backwards compatibility. It could be nice just to specify a single `jwks_pair` instead but I think the backwards compatibility was more important.

Reply:
+1 on that decision